lib/pure/collections/intsets.nim

```diff
@@ -251,6 +253,24 @@ proc incl*(s: var IntSet, other: IntSet) =
 
   for item in other: incl(s, item)
 
+proc toIntSet*(x: openArray[int]): IntSet {.since: (1, 3).} =
```

Comment:
```suggestion
proc toIntSet*(x: openArray[SomeSignedInt]): IntSet {.since: (1, 3).} =
```

Is `SomeSignedInt` preferred here?

Then users on 64-bit systems could write e.g. `[7'i32, 8, 9].toIntSet`, which might be reasonable to support given that they can currently write e.g. `myintset.incl(7'i32)` (and similar for `contains`, `containsOrIncl`, `excl` and `missingOrExcl`).

For such users, note that e.g. `myintset.incl(7'i64)` doesn't compile, so `[7'i64, 8, 9].toIntSet` wouldn't work either way.